pkg/agent/openflow/packetin.go

```diff
@@ -78,9 +78,11 @@ const (
 
 	// We use OpenFlow Meter for packetIn rate limiting on OVS side.
 	// Meter Entry ID.
-	PacketInMeterIDNP  = 1
-	PacketInMeterIDTF  = 2
-	PacketInMeterIDDNS = 3
+	// 1-255 are reserved for Egress QoS. The Egress QoS meterID leverage the same
+	// number as the mark allocated to the EgressIP.
+	PacketInMeterIDNP  = 256
+	PacketInMeterIDTF  = 257
+	PacketInMeterIDDNS = 258
```

Comment:
Not sure if there is any side-affect with this kind of ID change, @wenyingd could you help to review this PR and confirm?  Thanks.

Reply:
I am not worried about this change. Upgrading Antrea means that the Antrea Pod is recreated and everything is restarted. ovs-vswitchd will have empty tables and the old meters should not exist any more.